mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -365,7 +364,6 @@ def get_cmd(
     model_uri: str, port: int = None, host: int = None, timeout: int = None, nworkers: int = None
 ) -> Tuple[str, Dict[str, str]]:
     local_uri = path_to_local_file_uri(model_uri)
-    timeout = timeout or 60
```

Comment:
I think the following edit is necessary since `timeout` is `None` by default, which would break line 370:

```
timeout = timeout or MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT.get()
```

Reply:
I've removed that because the code calling it is either from 
evaluate for model
https://github.com/zencity/mlflow/blob/6f2231b3cd4be7ad8558d4a61e4220668e2529c6/mlflow/pyfunc/__init__.py#L576 (yes I've found another entry point to the scoring server)

Or the inner pyspark udf 

https://github.com/zencity/mlflow/blob/6f2231b3cd4be7ad8558d4a61e4220668e2529c6/mlflow/pyfunc/__init__.py#L1080

Or from

https://github.com/zencity/mlflow/blob/6f2231b3cd4be7ad8558d4a61e4220668e2529c6/mlflow/pyfunc/backend.py#L162

I'd rather keep it outside, so we'll support future additions to other scoring backends, mlserver supporting timeout (currently doesn't) etc. 

if not, let's put None, in all of those places, and add the line you suggested, but I don't think it's future proof

Reply:
@dbczumar

Reply:
Hi @alonisser , I'd rather put None in other places and put the line I suggested here. The reason is that, otherwise, if this method is called with the default `timeout=None`, it will break. It should not be the responsibility of each call site to supply a valid value of this argument else face an error, since this argument is optional.

Reply:
Done, I've kept two places also in the calling sites (removed one) as the original was hard coded to 60 and reverting to that is problematic. So we're both explicit in the calling site and ensured inside the scoring server logic @dbczumar

Reply:
@dbczumar